pulsar-common/src/main/java/org/apache/pulsar/common/policies/data/PersistentOfflineTopicStats.java

```diff
@@ -55,8 +55,8 @@
     public PersistentOfflineTopicStats(String topicName, String brokerName) {
         this.brokerName = brokerName;
         this.topicName = topicName;
-        this.dataLedgerDetails = Lists.newArrayList();
-        this.cursorDetails = Maps.newHashMap();
+        this.dataLedgerDetails = new ArrayList<>();
```

Comment:
I prefer this version (as probably the JVM deals better with having the constructor call insteadof havingan indirection), but we are using these helpers everywhere.
Probably we should ask on dev@ about this style change.

Reply:
I think this was already covered in the discussion here: https://lists.apache.org/thread.html/re99b342765b2cfb26f3da0eedfb78e7312de158bf6c7ed0085d32781%40%3Cdev.pulsar.apache.org%3E

Reply:
Correct.
Thanks for the pointer